calco.css

```diff
@@ -0,0 +1,49 @@
+*, ::before,*::after {
```

Comment:
sir please if you see my assigmen told me because i don't no why it is in changed file